src/main/java/uk/gov/hmcts/ccd/domain/service/casedataaccesscontrol/RoleAssignmentsMapper.java

```diff
@@ -0,0 +1,14 @@
+package uk.gov.hmcts.ccd.domain.service.casedataaccesscontrol;
+
+import org.mapstruct.Mapper;
+import org.mapstruct.factory.Mappers;
+import uk.gov.hmcts.ccd.data.casedataaccesscontrol.RoleAssignmentResponse;
+import uk.gov.hmcts.ccd.domain.model.casedataaccesscontrol.RoleAssignments;
+
+@Mapper(componentModel = "spring")
```

Comment:
Correct me If I am wrong: 
     I think, we are doing this, so componentModel should be componentModel = 'default'
    below is from the documentation of @Mapper:
     * <li> {@code default}: the mapper uses no component model, instances are
     * typically retrieved via {@link Mappers#getMapper(Class)}</li>